src/components/molecules/ScheduleRoomEvents/ScheduleRoomEvents.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import { PersonalizedVenueEvent } from "types/venues";
+
+import { WithVenueId } from "utils/id";
+
+import { ScheduleEvent } from "../ScheduleEvent";
+
+import "./ScheduleRoomEvents.scss";
+
+export interface ScheduleRoomEventsProps {
+  events: WithVenueId<PersonalizedVenueEvent>[];
+  scheduleStartHour: number;
+  isUsers?: boolean;
+}
+
+export const ScheduleRoomEvents: React.FC<ScheduleRoomEventsProps> = ({
+  events,
+  scheduleStartHour,
+  isUsers,
+}) => {
+  return (
+    <div className="RoomEvents">
+      {events.map((event, index) => (
+        <ScheduleEvent
+          key={`event-${event.id}-${index}`}
+          isUsers={isUsers}
+          event={event}
+          scheduleStartHour={scheduleStartHour}
+        />
+      ))}
```

Comment:
memo

Reply:
Or can maybe `React.memo` the entire component potentially?

Why are we using `index` as part of the `key` here? Is `event.id` not unique? We generally don't want to ever be using `index` in the `key` as it's not stable/causes rendering issues and similar

Reply:
removed index: https://github.com/sparkletown/sparkle/pull/1302/commits/3773b544eaa9b24083615b54b444167ca724eff2 
but haven't applied memo yet.